lago/vm.py

```diff
@@ -120,13 +120,10 @@ def bootstrap(self):
                     self.vm._spec['disks'][0]['path'],
                     [
                         sysprep.set_hostname(self.vm.name()),
-                        sysprep.set_root_password(self.vm.root_password()),
```

Comment:
I think we should keep this if it doesn't have any significant time penalty:
Current use case: when using plain qcow files locally(without lago-templates)
Future use case: using other image providers.
overall, I think we should relay less on 'baked' templates and do the logic inside if we want to drop lago-images one day.